package.json

```diff
@@ -25,7 +25,7 @@
     "stylelint": "15.2.0",
     "stylelint-config-css-modules": "4.2.0",
     "stylelint-config-standard": "30.0.1",
-    "turbo": "latest",
+    "turbo": "^1.8.6",
```

Comment:
@nathan-ong curious why we had to make this change. Did you run `pnpm install` instead of `npm install`?

Reply:
Ahh, yeah my linter was bugging out. I think I may have resolved it, so I'll revert this